graalvm-nativeimage/quarkus/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+
+quarkus.datasource.db-kind=oracle
+quarkus.datasource.jdbc.driver=oracle.jdbc.driver.OracleDriver
+quarkus.datasource.jdbc.url=jdbc:oracle:thin:@gdpaulsep27i_tp?TNS_ADMIN=/Users/pparkins/Downloads/Wallet_gdpaulsep27o
+quarkus.datasource.username=inventoryuser
+quarkus.datasource.password=Welcome12345
```

Comment:
Wallet password here

Reply:
fixed it (a lot of these are just templates/placeholders btw)